adapter_files/precice.cpp

```diff
@@ -478,6 +483,8 @@ double Precice::advance(double computedTimestepLength) {
       // Load Ramping functionality: Reduce force vector before transferring by a ramping factor, which increases with
       // the number of elapsed time steps; Achtung: ExtIter beginnt bei 0 (ohne Restart) und bei einem Restart
       // (Startlösung) nicht bei 0, sondern bei der Startiterationsnummer
+	  
+	  /* Temporarily commenting out - TODO
```

Comment:
How temporarily? :innocent: 
Why?

Reply:
Probably not something I will work on in the near future, might be useful keeping the old code lines there, but they definitely don't work for the newest version of SU2. Would you suggest I make that more clear or just remove it altogether?